### PR TITLE
UI component: SAM Header + Footer Assets

### PR DESCRIPTION
**Pull request by eoladipo**
@joshbruce 

- Added the footer and header sass files in the components style sheet folder
- Added iae-global sass file in the core folder; it’s comprised of global styles used across iae components
- Added a margin-padding sass file in the core folder; it’s a helper stylesheet that standardizes how margins and paddings are applied
- Updated “all.scss" file to import the added files
- Added Transition Sam image into the image folder in the assets directory

### CODE REVIEW

File: src/stylesheets/core/_margin-padding.scss [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+.p_L-5x { @extend %padding-left-5x; }
+.p_L-6x { @extend %padding-left-6x; }
+.p_L-7x { @extend %padding-left-7x; }
+.p_L-8x { @extend %padding-left-8x; }
+.p_R-2x { @extend %padding-right-2x; }
+.p_R-3x { @extend %padding-right-3x; }
+.p_R-4x { @extend %padding-right-4x; }
+.p_R-5x { @extend %padding-right-5x; }
+.p_R-6x { @extend %padding-right-6x; }
+
```
Comment (joshbruce) [inline src/stylesheets/core/_margin-padding.scss]: If you don't have time, no worries, curious if we can make the class names more human readable?

Putting it in user story terms: As a front-end developer, when I look at the HTML, I should be able to understand what is being accomplished by the class.

File: src/stylesheets/core/_iae-global.scss [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+}
+
+@mixin transition($args...) {
+    -webkit-transition: $args;
+    -moz-transition: $args;
+    -ms-transition: $args;
+    -o-transition: $args;
+    transition: $args;
+}
+
```
Comment (joshbruce) [inline src/stylesheets/core/_iae-global.scss]: Can we move this @mixin to `/src/stylesheets/_utilities.scss`?

Curious: Does this duplicate the Bourbon method? http://bourbon.io/docs/#transitions

File: src/stylesheets/core/_iae-global.scss [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+@mixin opacity($opacity) {
+    opacity: $opacity;
+    $opacity-ie: $opacity * 100;
+    filter: alpha(opacity=$opacity-ie); //IE8
+}
+
```
Comment (joshbruce) [inline src/stylesheets/core/_iae-global.scss]: Can we move this @mixin to `/src/stylesheets/_utilities.scss`?

**Comment by burgwyn** [comment]
Moved three mixins to `/src/stylesheets/_utilities.scss`
Added comments on `/src/stylesheets/margin-and-padding-spacing.scss`

File: src/stylesheets/core/_margin-padding.scss [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+     %padding-vertical-#{$i}x{ padding-bottom: $spacing * $i !important; }
+
+     %padding-left-#{$i}x,
+     %padding-horizontal-#{$i}x { padding-left: $spacing * $i !important; }
+
+     %padding-right-#{$i}x,
+     %padding-horizontal-#{$i}x{ padding-right: $spacing * $i !important; }
+    }
+}
+
```
Comment (joshbruce) [inline src/stylesheets/core/_margin-padding.scss]: Can we move this to `_utilities.scss`.

**Pull request merged**